content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,258 @@
+    {{< /text >}}
+
+1. Check the log and verify:
+
+    * The log includes an `envoy.filters.http.rbac` filter to enforce the authorization policy
+      on each incoming request.
+    * Istio updates the filter accordingly after you update your authorization policy.
+
+1. For example, you might see something similar to the following:
+
+    {{< text plain >}}
+    {
+     "name": "envoy.filters.http.rbac",
+     "config": {
+      "rules": {
+       "policies": {
+        "productpage-viewer": {
+         "permissions": [
+          {
+           "and_rules": {
+            "rules": [
+             {
+              "or_rules": {
+               "rules": [
+                {
+                 "header": {
+                  "exact_match": "GET",
+                  "name": ":method"
+                 }
+                }
+               ]
+              }
+             }
+            ]
+           }
+          }
+         ],
+         "principals": [
+          {
+           "and_ids": {
+            "ids": [
+             {
+              "any": true
+             }
+            ]
+           }
+          }
+         ]
+        }
+       }
+      },
+      "shadow_rules": {
+       "policies": {}
+      }
+     }
+    },
+    {{< /text >}}
+
+    It means the proxy of productpage has enabled the `envoy.filters.http.rbac` filter with rules
+    that allows anyone to access it via `GET` method. The `shadow_rules` are not used and you can
+    ignored them safely.
+
+## Ensure Proxies Enforce Policies Correctly
+
+Proxies eventually enforce the authorization policies. The following steps help you ensure the proxy
+is working as expected:
+
+> Note: The command used in this section assumes you have deployed [Bookinfo application](/docs/examples/bookinfo/).
+otherwise you should replace `"-l app=productpage"` with your actual pod.
+
+1. Turn on the authorization debug logging in proxy with the following command:
+
+    {{< text bash >}}
+    $ kubectl exec  $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- curl localhost:15000/logging?rbac=debug -s
+    {{< /text >}}
+
+1. Verify you see the following output:
+
+    {{< text plain >}}
+    active loggers:
+      ... ...
+      rbac: debug
+      ... ...
+    {{< /text >}}
+
+1. Visit the `productpage` in your browser to generate some logs.
+
+1. Print the proxy logs with the following command:
+
+    {{< text bash >}}
+    $ kubectl logs $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy
+    {{< /text >}}
+
+1. Check the output and verify:
+
+    * The output log shows either `enforced allowed` or `enforced denied` depending on whether the request
+      was allowed or denied respectively.
+
+    * Your authorization policy expects the data extracted from the request.
+
+1. You should see something similar to the following:
```

Comment:
Replace this step with:
1. The following output means there is a `GET` request at path `/productpage` and the policy allows the request.
    The `shadow denied` has no effect and you can ignore it safely.